bundle/edu.gemini.ags.client.impl/src/main/scala/edu/gemini/ags/client/impl/QueryArgs.scala

```diff
@@ -56,6 +56,7 @@ object QueryArgs {
     case g: GracesBlueprint     => Right(Instrument.GmosNorth.id) // REL-3288: GRACES uses GMOS-N OI.
     case g: TexesBlueprint      => Right(Instrument.Nifs.id)      // REL-1062
     case g: VisitorBlueprint    => Right(Instrument.Niri.id)      // REL-1090
+    case g: ZorroBlueprint      => Right(Instrument.Nifs.id)      // REL-3454: treat Zorro like Alopeke and DSSI.
```

Comment:
Why is that we treat them like `NIFS`? i probably did this myself 😄

Reply:
It's something about P1 (or possibly P2?) not being properly configurable with them now, so they default by request of science to be the same as `Nifs`.